domains/primitives/messenger/src/lib.rs

```diff
@@ -219,6 +230,9 @@ sp_api::decl_runtime_apis! {
 
         /// Returns storage key for channels for given chain and channel id.
         fn channel_storage_key(chain_id: ChainId, channel_id: ChannelId) -> Vec<u8>;
+
+        /// Returns all the open channels to other chains.
+        fn open_channels() -> BTreeSet<(ChainId, ChannelId)>;
```

Comment:
I'm not sure when we're going to reset the API versions again, some of them might be around for a while after mainnet launch:
```suggestion
        /// Returns all the open channels to other chains.
        /// Only in API versions 2 and later.
        fn open_channels() -> BTreeSet<(ChainId, ChannelId)>;
```

Reply:
Our code already handles the api version through checks, no reason to add it here.
If and when we reset the api versions, we have TODOs to remove the compatibility code as well.